Calculator.php

```diff
@@ -15,25 +15,10 @@
 
     $command = choose('Enter command: ', AVAILABLE_COMMANDS);
 
-    if ($command == INFO) {
-        info((implode(' ;  ', AVAILABLE_COMMANDS)) . ' ;');
+    $isNoMathFunction = in_array($command, [HISTORY,INFO,QUIT]);
```

Comment:
let's create the SYSTEM_COMMANDS const

```suggestion
    $isSystemCommand = in_array($command, SYSTEM_COMMANDS);
```